tests/aws/services/sns/test_sns.snapshot.json

```diff
@@ -5078,5 +5078,9 @@
         ]
       }
     }
+  },
+  "tests/aws/services/sns/test_sns.py::TestSNSSubscriptionSQSFifo::test_message_to_fifo_sqs_ordering": {
+    "recorded-date": "19-02-2025, 01:29:15",
+    "recorded-content": {}
```

Comment:
empty snapshot is because we're using a transformer fixture with `autouse`, so it creates an entry even if no snapshot are recorded 😅